File: 705-design-hashset/705-design-hashset.py
class MyHashSet:
    
    hash = []

    def __init__(self):
        self.hash = {}
        

    def add(self, key: int) -> None:
        self.hash[key] = 1
        

    def remove(self, key: int) -> None:
        if self.hash.get(key):
            self.hash.pop(key)
        

    def contains(self, key: int) -> bool:
        return self.hash.get(key, False)
        


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)

File: 705-design-hashset/705-design-hashset.ts
class MyHashSet {
    hash = {};
    constructor() {
        this.hash = {}
    }

    add(key: number): void {
        this.hash[key] = 1;
    }

    remove(key: number): void {
        delete this.hash[key];
    }

    contains(key: number): boolean {
        return this.hash[key] ? true : false;
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * var obj = new MyHashSet()
 * obj.add(key)
 * obj.remove(key)
 * var param_3 = obj.contains(key)
 */